installer.nu

```diff
@@ -9,93 +9,96 @@ let nupac_repo_cache = $"https://raw.githubusercontent.com/skelly37/nupac/($defa
 let nupac_module = $"https://raw.githubusercontent.com/skelly37/nupac/($default_branch)/modules/nupac/nupac.nu"
 let nupac_json = $"https://raw.githubusercontent.com/skelly37/nupac/($default_branch)/modules/nupac/nupac.json"
 
-def scripts-path [] {
+let nupac_base_path = (
     $env
     |get --ignore-errors "NUPAC_DEFAULT_LIB_DIR"
     |default (
         $nu.config-path
         |path dirname
         |path join "nupac"
     )
-}
-
-mkdir (
-    (scripts-path)
-    |path join "nupac"
+)
+let packages_path = (
+    $nupac_base_path
+    |path join 'packages'
 )
 
-# nupac index
+# # nupac index
 let nu_pkgs = (
-    (scripts-path)|
-    path join "nu-pkgs.nu"
+    $nupac_base_path
+    |path join "nu-pkgs.nu"
 )
-
 let repo_cache_path = (
-    (scripts-path)|
-    path join "repo-cache.json"
+    $nupac_base_path
+    |path join "repo-cache.json"
 )
 
-let nupac_path = (
-    (scripts-path)
-    |path join "nupac/nupac.nu"
-    |into string
+let nupac_module_path = (
+    $packages_path
+    |path join 'nupac'
+)
+let nupac_script_path = (
+    $nupac_module_path
+    |path join 'nupac.nu'
 )
 
 let nupac_json_path = (
-    (scripts-path)
-    |path join "nupac/nupac.json"
-    |into string
+    $nupac_module_path
+    |path join 'nupac.json'
 )
+$nu.scope.vars|select name value|sort-by value
+mkdir $nupac_base_path $packages_path $nupac_module_path
```

Comment:
```suggestion
mkdir $nupac_module_path
```

No need to use mkdir thrice, as `$nupac_module_path` contains all of them

Reply:
I wanted to be verbose but good point